job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml

```diff
@@ -102,7 +102,7 @@
               string(name: 'ICON', value: ':alarm_clock:'),
               string(name: 'STAGE', value: "${ENVIRONMENT}"),
               string(name: 'STATUS', value: 'FAILED'),
-              string(name: 'CHANNEL', value: '#dm-2ndline'),
+              string(name: 'CHANNEL', value: ENVIRONMENT == 'production' ? '#dm-2ndline' : '#dm-release'),
```

Comment:
Just to check, are we using ternary here rather than the usual jinja logic `{{'#dm-2ndline' if environment == 'production' else '#dm-release'}}` as we don't have `environment` available to us?

Reply:
Err no, I just forgot we had jinja available to us... it would probably be better to do it that way if poss.